PocketKit/Sources/SharedPocketKit/UserDefaultsKey.swift

```diff
@@ -24,6 +24,9 @@ public extension UserDefaults {
         case listSelectedSortForSaved = "listSelectedSortForSaved"
         case listSelectedSortForArchive = "listSelectedSortForArchive"
         case readerFontSizeAdjustment = "readerFontSizeAdjustment"
+        case readerScaleFactorIndex = "readerScaleFactorIndex"
+        case readerMarginsIndex = "readerMarginsIndex"
+        case readerTextAlignment = "readerTextAlignment"
```

Comment:
Every time I see these, I think this was a nice, relatively simple change to our codebase that made `UserDefaults` usage a little cleaner.